alf/utils/normalizers.py

```diff
@@ -171,6 +173,12 @@ def _normalize(m2, t, m=None):
             return alf.nest.map_structure(_normalize, self._m2_averager.get(),
                                           tensor)
 
+    def forward(self, input):
+        if self.training:
```

Comment:
Will this self.training flag automatically be set?

Reply:
Yes.